CharlyUwampojejeCV.css

```diff
@@ -0,0 +1,60 @@
+body{
+    background-image: linear-gradient(#636162, #b1afaf);
+    height: 122vh;
+    letter-spacing: 1px;
+}
+*{
+    margin: 0px; padding: 0px; font-family: 'montserrat', century gothic;
+}
+.main{
+    background-color: white;
+    height: auto;
+    width: 700px;
+    position: absolute;
+    transform: translate( -50%, -50%);
+    left: 50%;
+    top: 60%;
+}
+.left{
+    background-image: repeating-linear-gradient(268deg, rgba(255,255,255, 0) 0px, rgba(255,255,255, 0) 2px,transparent 2px, transparent 4px),linear-gradient(268deg, rgba(240,240,240, 0),rgba(59,59,59, 0.78));
+    width: 240px;
+    float: left;
+    text-align: center;
+    font-size: 90%;
+    padding: 10px;
+}
+
+.profile-img{
+    height: 180px;
+    width: 180px;
+    border-radius: 60%;
+    margin-bottom: 30px;
+    margin-left: 10px;
+}
+.profile-img img{
+    height: 200px;
+    width: 200px;
+    border-radius: 100px;
+
+}
+.right{
+    font-size: 70%;
+    padding: 0px 30px;
+}
+
+.name-div{
+    padding:70px 0px 60px 55px;
+    text-align: start;
+    letter-spacing: 2px;
+    font-size: large;
+}
+.nom{
```

Comment:
Change the class name. It has to be in english.

Reply:
Done